src/app/(proper_react)/redesign/(authenticated)/user/dashboard/Dashboard.stories.tsx

```diff
@@ -151,7 +151,13 @@ const DashboardWrapper = (props: DashboardWrapperProps) => {
   };
 
   return (
-    <Shell l10n={getEnL10nSync()} session={mockedSession} nonce="">
+    <Shell
+      l10n={getEnL10nSync()}
+      session={mockedSession}
+      nonce=""
+      monthlySubscriptionUrl={""}
+      yearlySubscriptionUrl={""}
```

Comment:
Nit: No need for the curly braces. This could be just
```suggestion
      monthlySubscriptionUrl=""
      yearlySubscriptionUrl=""
```